bokehjs/src/coffee/models/widgets/defaults.coffee

```diff
@@ -0,0 +1,204 @@
+
+all_defaults = {
```

Comment:
Actually I should have suggested in the issue you do `git clean -dfx` before any work.